libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -303,8 +309,8 @@ private void buildUserIdentityBasedOnProfileType(
                 successfulVcs.add(vc);
             }
         }
-        if (Vot.P0.equals(vot)) {
-            userIdentityBuilder.returnCode(getFailReturnCode(contraIndicators));
+        if (Vot.P0.equals(achievedVot)) {
```

Comment:
I believe we want to leave open the option of someone requesting a P1 but actually getting a P2 and treating that as success.

Reply:
Yeah, if possible we'd like to be able to handle a VTR of `[P1, P2]` correctly, even if it's not going to be part of the MVP.

In that scenario, it could be possible to have `targetVot = P1` but `achievedVot = P2`. However the inverse should _not_ be possible - i.e. we should never see `targetVot = P2` and `achievedVot = P1` (you should drop to `P0` if you don't achieve the target).